src/styles/blocks/header.scss

```diff
@@ -1,3 +1,5 @@
+$link-color: #00acdc;
```

Comment:
same here regarding the name for a variable. It is usually `$primary-green`, `$secondary-green`, or sth like that